src/servlets/utils/helpers.ts

```diff
@@ -47,6 +57,65 @@ export const getImageUrl = (cid: string, gateway: string | null): string => {
   return `${gateway}${cid}`
 }
 
+type ExploreInfoType = {
+  title: string,
+  description: string,
+  image: string
+}
+export const getExploreInfo = (type: string): ExploreInfoType  => {
+  const imageMap: { [key: string]: ExploreInfoType } = {
+    'heavy-rotation': {
+      title: 'Heavy rotation',
+      description: 'Your top tracks, in one place',
+      image: HEAVY_ROTATION_URL
+    },
+    'let-them-dj': {
+      title: 'Let them dj',
+      description: 'Playlists created by the people you follow',
+      image: LET_THEM_DJ_URL
+    },
+    'best-new-releases': {
+      title: 'Best new releases',
+      description: 'From the artists you follow',
+      image: BEST_NEW_RELEASES_URL
+    },
+    'under-the-radar': {
+      title: 'Under the radar',
+      description: 'Tracks you might have missed from the artists you follow',
+      image: UNDER_THE_RADAR_URL
+    },
+    'top-albums': {
+      title: 'Top albums',
+      description: 'The top albums from all of Audius',
+      image: TOP_ALBUMS_URL
+    },
+    'top-playlists': {
+      title: 'Top playlists',
+      description: 'The top playlists on Audius right now',
+      image: TOP_PLAYLISTS_URL
+    },
+    'most-loved': {
+      title: 'Most loved',
+      description: 'Tracks favorited by the people you follow',
+      image: MOST_LOVED_URL
+    },
+    'feeling-lucky': {
+      title: 'Feeling lucky',
+      description: 'A purely random collection of tracks from Audius',
+      image: FEELING_LUCKY_URL
+    },
+  }
+  if (!Object.keys(imageMap).includes(type)) {
+    return {
+      title: 'Just for you',
+      description: 'Content curated for you based on your likes, reposts, and follows.' +
```

Comment:
Nit, but multi line string with tick marks is usually what we tend to do https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Maybe we could standardize this in our style guide :)